includes/class-activitypub.php

```diff
@@ -707,16 +707,54 @@ public static function updated_postmeta( $meta_id, $object_id, $meta_key, $meta_
 	 * Register user meta.
 	 */
 	public static function register_user_meta() {
+		$blog_prefix = $GLOBALS['wpdb']->get_blog_prefix();
+
 		\register_meta(
 			'user',
-			'activitypub_also_known_as',
+			$blog_prefix . 'activitypub_also_known_as',
 			array(
 				'type'              => 'array',
 				'description'       => 'An array of URLs that the user is known by.',
-				'single'            => false,
+				'single'            => true,
 				'default'           => array(),
 				'sanitize_callback' => array( Sanitize::class, 'url_list' ),
 			)
 		);
+
+		\register_meta(
+			'user',
+			$blog_prefix . 'activitypub_description',
+			array(
+				'type'              => 'string',
+				'description'       => 'The user&#8217;s description.',
```

Comment:
This entity escaping doesn't make a ton of sense in a non-translated string that won't get displayed. How do I best allow for curly quotes in these circumstances?

Reply:
this is UTF-8 we can use them as ASCII